src/main/java/hu/bme/mit/spaceship/TorpedoStore.java

```diff
@@ -14,6 +14,7 @@
 
   private int torpedoCount = 0;
 
+  private Random generator = new Random();
```

Comment:
Approved. This fixes the bug identified by sonarqube and won't reuse the Random object in each function call.